platform/mellanox/mlnx-platform-api/sonic_platform/chassis.py

```diff
@@ -29,6 +28,7 @@
 MLNX_NUM_PSU = 2
 
 GET_HWSKU_CMD = "sonic-cfggen -d -v DEVICE_METADATA.localhost.hwsku"
+GET_PLATFORM_CMD = "sonic-cfggen -d -v DEVICE_METADATA.localhost.platform"
```

Comment:
@Junchao-Mellanox why not to use `from sonic_device_util import get_machine_info`?

Reply:
Good point! I didn't notice that there is such a convinient utilitiy class. Will change soon.

Reply:
Fixed